build/configs/qemu/README.md

```diff
@@ -75,3 +75,98 @@ Enter `Board Selection` -> `Build for qemu hardware`
 Select `NONE`(64KB) or `Build for SRAM increased QEMU Hardware`(16MB)  
 Enter `Chip Selection` -> `Boot Memory Configuration`  
 Set `Primary RAM size` to `65536`(64KB) or `16777216`(16MB)
+
+##Steps to verify network stack in QEMU is as follows
+build qemu as follow
+    a) cd os/tools
+    b) ./configure.sh qemu/tc_16m
+    c) cd ..
+    d) make
+
+Connect WiFi AP to Linux machine(wired connection. dhcp should be
+enabled in the AP. Then DHCP server will be running in AP)
+
+Create bridge and tap interface by adding below commands into shell
+script file(net_up.sh)
+
+        #!/bin/sh
+        #
+        # script to bring up the tun device in QEMU in bridged mode
+        # first parameter is name of tap device (e.g. tap0)
+        #
+        # some constants specific to the local host - change to suit your host
+        #
+        ETH0IPADDR=192.168.1.20
+        MASK=255.255.255.0
+        GATEWAY=192.168.1.1
+        BROADCAST=192.168.1.255
+        #
+        # First take eth0 down, then bring it up with IP address 0.0.0.0
+        #
+        /sbin/ifdown enp3s0
+        /sbin/ifconfig enp3s0 0.0.0.0 promisc up
+        #
+        # Bring up the tap device (name specified as first argument, by QEMU)
+        #
+	/usr/sbin/openvpn --mktun --dev $1 --user `id -un`
+        /sbin/ifconfig $1 0.0.0.0 promisc up
+        #
+        # create the bridge between eth0 and the tap device
+        #
+        /sbin/brctl addbr br0
+        /sbin/brctl addif br0 enp3s0
+        /sbin/brctl addif br0 $1
+        #
+        # only a single bridge so loops are not possible, turn off spanning tree protocol
+        #
+        /sbin/brctl stp br0 off
+        #
+        # Bring up the bridge with ETH0IPADDR and add the default route
+        #
+        /sbin/ifconfig br0 $ETH0IPADDR netmask $MASK broadcast $BROADCAST
+        /sbin/route add default gw $GATEWAY
+        #
+        # stop firewall - comment this out if you don't use Firestarter
+        #
+        /sbin/service firestarter stop
+
+execute the shell script with tap0 as argument(ex: sudo ./net_up.sh tap0)
+
+Execute Qemu with netwrok interface as below
+        sudo qemu-system-arm -M lm3s6965evb -kernel ../build/output/bin/tinyara.bin -nographic -net nic -net tap,ifname=tap0,script=no
+
+Bringup Interface
+        ifup en1
+Get IP from AP
+        ifconfig en1 dhcp
+Now Ping to Host IP(192.xxx.x.xxx) and ping from Host to
+Qemu(192.xxx.x.xxx)
+
+To bring down tap0 and bridge interfaces execute add the below cmd into
+shell scripte(net_down.sh) and execute shell script.
```

Comment:
can you specify the parameters here, for example net_up.sh tap0, so that one understands the uage well? Thanks.

Reply:
I will add it